openapi_server/apis/users_api.py

```diff
@@ -33,31 +37,49 @@
     "/users",
     responses={
         200: {"model": User, "description": "successful operation"},
+        409: {"description": "Username already exists"},
     },
     tags=["users"],
     summary="Create user",
     response_model_by_alias=True,
 )
 async def create_user(
     create_user: CreateUser = Body(None, description="Created user object"),
-) -> User:
+) -> Union[User, Response]:
     """"""
+    try:
+        DbUser.get(create_user.username)
+        return Response(status_code=409)
+    except Exception as e:
+        pass
```

Comment:
Could we make response with 500 error? and for the body, we can probably define the format { message: e.message(check if e.message is correct or else), error: e }